src/test/java/com/sonyericsson/jenkins/plugins/bfa/PluginImplHudsonTest.java

```diff
@@ -40,23 +40,33 @@
 import com.sonyericsson.jenkins.plugins.bfa.test.utils.DifferentKnowledgeBase;
 import hudson.ExtensionList;
 import net.sf.json.JSONObject;
-import org.jvnet.hudson.test.HudsonTestCase;
+import org.junit.Rule;
+import org.junit.Test;
+//import org.jvnet.hudson.test.HudsonTestCase;
+import org.jvnet.hudson.test.JenkinsRule;
 import org.kohsuke.stapler.StaplerRequest;
 import org.powermock.reflect.Whitebox;
 
 import java.util.List;
 
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
 import static org.mockito.Matchers.eq;
 import static org.mockito.Matchers.isA;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 /**
- * {@link HudsonTestCase}s for {@link PluginImpl}.
+ * TestCases for {@link PluginImpl}.
  *
  * @author Robert Sandell &lt;robert.sandell@sonyericsson.com&gt;
  */
-public class PluginImplHudsonTest extends HudsonTestCase {
+public class PluginImplHudsonTest {
+    @Rule
+    private JenkinsRule jenkins = new JenkinsRule();
```

Comment:
Example of how to solve the checkstyle public problem https://github.com/jenkinsci/gerrit-trigger-plugin/blob/master/src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/spec/ParameterModeJenkinsTest.java#L53